### PR TITLE
Add configuration for setuptools 

### PR DESCRIPTION
**Pull request by AntonKarabaza**
Add setup.py with major project information for distribution.
Update README.md with instructions how to install the package as Python
dependency.
Update .travis.yml to install the package after verify_project.sh ran.

Affects #4

### CODE REVIEW

**Review by ivan-podorozhnyi — CHANGES_REQUESTED** [review]
@AntonKarabaza I suggest its better to use script to read dependencies in`install-required`field from `requirements.txt`.
Check: https://github.com/ets-labs/python-dependency-injector/blob/master/setup.py - it looks good. 
Also I think `def read(fname):` can be replaced with similar script (you can find it there too). But second is up to you, you can accept my idea or keep yours.

**Comment by AntonKarabaza** [comment]
@ivan-podorozhnyi Please, take a look now.

**Review by ivan-podorozhnyi — APPROVED** [review]
@AntonKarabaza Looks good, thank you!

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+with open("README.md") as readme_file:
+    description = readme_file.read()
+
+with open("requirements.txt") as requirements_file:
+    requirements = requirements_file.readlines()
+
+setup(
+    name="demo_at",
+    version="0.1",
```
Comment (extsoft) [inline setup.py]: Use Semantic versioning here. Set `0.1.0` version.

**Review by extsoft — CHANGES_REQUESTED** [review]

**Comment by AntonKarabaza** [comment]
@extsoft Versioning is changed to Semantic and now has the form `0.1.0`.

**Comment by AntonKarabaza** [comment]
@extsoft Before merge this pull request #18 have to be merged.

**Comment by extsoft** [comment]
> @extsoft Before merge this pull request #18 have to be merged.

@AntonKarabaza done.

**Comment by AntonKarabaza** [comment]
@extsoft Branch rebased and ready to merge.

**Review by extsoft — APPROVED** [review]

**Pull request merged**